projects/cdk/services/theme-color.service.ts

```diff
@@ -0,0 +1,32 @@
+import {DOCUMENT} from '@angular/common';
+import {inject, Injectable} from '@angular/core';
+import {tuiCreateToken} from '@taiga-ui/cdk/utils';
+
+export const TUI_THEME_COLOR = tuiCreateToken<string | null>(null);
+
+@Injectable({
+    providedIn: 'root',
+})
+export class TuiThemeColorService {
+    private readonly doc = inject(DOCUMENT);
+    private readonly initial = inject(TUI_THEME_COLOR);
+
+    constructor() {
+        if (this.initial) {
+            this.color = this.initial;
+        }
+    }
+
+    public set color(color: string) {
+        let meta = this.doc.querySelector('meta[name="theme-color"]');
+
+        if (!meta) {
+            meta = this.doc.createElement('meta');
+            meta.setAttribute('name', 'theme-color');
+            this.doc.head.appendChild(meta);
```

Comment:
Could we use just built-in `Meta`-service ?
https://angular.dev/api/platform-browser/Meta

Do we need actually need `TuiThemeColorService` or can just add example with `Meta` ?

Reply:
fixed